src/NosCore.Handler/CommandPacketHandler.cs

```diff
@@ -29,13 +33,45 @@ public CommandPacketHandler(ClientSession session)
         #region Methods
         public void Speed(SpeedPacket speedPacket)
         {
-            if (speedPacket.Speed >= 60)
+            if (speedPacket.Speed > 0 || speedPacket.Speed < 60)
             {
```

Comment:
Useless & not working condition

Reply:
thank you seems my post is displaying without specifing  the line :/ this  is the same thing i told 33min ago

Reply:
As well, something good for the user (imo) would be to set the speed at 59 even though he entered 999, instead of a return